platform/android/app/src/main/kotlin/coop/polypoly/polypod/polyOut/PolyOut.kt

```diff
@@ -157,7 +158,7 @@ open class PolyOut(
             }
         }
         val retVal = supervisorScope {
-            this.async {
+            this.async(Dispatchers.IO) {
```

Comment:
Yeah, `withContext` would have been cleaner, but I went with minimal fix.

I guess you mean `supervisorScope`, from briefly reading the documentation, it is related to how errors is handled. I did not investigate if this scope is really needed, this could be part of some improvement.